controllers/recruitersController.js

```diff
@@ -0,0 +1,19 @@
+const recruiterQuery = require('../models/recruiters')
+
+
+const addRecruiter = async (req, res) => {
+  try {
+    const timeStamp=req._startTime
+    const result = await recruiterQuery.addRecruiterInfo(req.body, req.params.username, timeStamp)
+    res.send(result)
```

Comment:
this should be `return res.send....`

Reply:
`return res.json....`
right? @ankurnarkhede

Reply:
Yes, @swarajpure

Reply:
Done!